src/App.css

```diff
@@ -0,0 +1,164 @@
+.twitter-app{
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  margin: 10px;
+}
+
+/* Side navbar*/
+.side-navbar{
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 0;
+  flex-grow: 1;
+  max-width: 230px;
+}
+
+.logo{
+  width: 46px;
+  height: 2rem;
+  padding-bottom: 7px;
+}
+
+.navbar-list-item{
+  padding: 12px;
+}
+.navbar-list-item:hover{
+  color: rgba(29,161,242,1.00);
+}
+
+.navbar-list-item-logo{
+  display: inline-flex;
+}
+.navbar-list-item-logo svg{
+  height: 1.6rem;
+}
+
+.navbar-list-item-text{
+  display: inline-flex;
+  vertical-align: 5px;
+}
+.title{
+  font-size: 20px;
+  font-weight: 550; 
+  margin-left: 10px;
+}
+
+/* Buttons */
+
+.tweet-btn{
+  background-color: rgb(29, 161, 242);
+  border: 0.1px solid rgb(10, 104, 161);
```

Comment:
Since you use the color values in many places. 
When you will use sass (scss files) you will see that adding colors and more common things as part of a separate file that you use as a base file is more convenient and easier to change and use